package.json

```
@@ -41,7 +43,9 @@
     "generate:test-certs": "./scripts/generate-test-certs.sh certs",
     "postversion": "git push && git push --tags",
     "dist:modules": "modclean -r -n default:safe && yarn install --production",
-    "predeploy": "npm-run-all generate:proxy-models build dist:modules"
+    "predeploy": "npm-run-all generate:proxy-models build dist:modules",
```

I we use a main `generate` yarn task ([comment](https://github.com/pagopa/io-backend/pull/757/files#r559688934)) maybe we need to modify this

```suggestion
    "predeploy": "npm-run-all generate build dist:modules",
```